_posts/2018-12-14-Bazel-0.20.0.md

```diff
@@ -0,0 +1,91 @@
+
+The Bazel team is happy to announce a new version of Bazel, [Bazel 0.20](https://github.com/bazelbuild/bazel/releases/tag/0.20.0).
+
+With this release, we communicate [breaking changes](https://docs.google.com/document/d/1Dj5PBLmPVg9ZyApm4GobM3y-mDgY3mVaqpRVttOe-ZQ/)
+following our [communication policy](https://docs.google.com/document/d/1q5GGRxKrF_mnwtaPKI487P8OdDRh2nN7jX6U-FXnHL0/).
+
+[Breaking changes in 0.20](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Abreaking-change-0.20):
+  - [--incompatible_remove_native_http_archive](https://github.com/bazelbuild/bazel/issues/6570)
+   - [--incompatible_remove_native_git_repository](https://github.com/bazelbuild/bazel/issues/6569)
+   - [--incompatible_disable_cc_toolchain_label_from_crosstool_proto](https://github.com/bazelbuild/bazel/issues/6434)
+   - [--incompatible_disable_depset_in_cc_user_flags](https://github.com/bazelbuild/bazel/issues/6384)
+   - [--incompatible_disable_cc_configuration_make_variables](https://github.com/bazelbuild/bazel/issues/6381)
+   - [--incompatible_disallow_conflicting_providers](https://github.com/bazelbuild/bazel/issues/5902)
+   - [--incompatible_range_type](https://github.com/bazelbuild/bazel/issues/5264)
+
+Changes for which 0.20 is a migration window are marked with [GitHub label 'migration-0.20'](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Amigration-0.20)
+ 
+Breaking changes in the next release (0.21 )[are marked with GitHub label 'breaking-change-0.21'](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+label%3Abreaking-change-0.21)
+
+## General Changes
+
+  - (Linux only) Bazel now shuts down when idle for 5 minutes and the system
+    is low on RAM.
+  - We're deprecating `--genrule_strategy=<value>` with the intent of removing it in early 2019; please use `--strategy=Genrule=<value>` instead
+    (see [#6760](https://github.com/bazelbuild/bazel/issues/6760)).
+  - Bazel now queryies remote server Capabilities (per REAPI v2)
+  - Use different server log files per Bazel server process; java.log
+    is now a symlink to the latest log.
+  - The code coverage report now includes the actual paths to header
+    files instead of the ugly, Bazel generated, virtual includes path.
+  - Bazel sorts remote Platform properties for remote execution. 
+
+## C++
+
+  - The --experimental_no_dotd_scanning_with_modules command line argument is not supported anymore.
+  - The --prune_cpp_modules command line option is not supported anymore.
+  - The --experimental_prune_cpp_input_discovery command line option is not supported anymore.
+  - All cc_toolchains depended on from cc_toolchain_suite.toolchains are now analyzed when not using
+    platforms in order to select the right cc_toolchain.
+  - CROSSTOOL file is now read from the package of cc_toolchain, not from
+    the package of cc_toolchain_suite. This is not expected to break anybody since
+    cc_toolchain_suite and cc_toolchain are commonly in the same package.
```

Comment:
all of the --snake_case_flags should be wrapped in backticks or they'll get treated as italics